### PR TITLE
make progress almost availalbe

### PR DESCRIPTION
**Pull request by jamesgorrie**
TODO:

- [x] Need to add markdown loader

### CODE REVIEW

File: catalogue/webapp/pages/progress.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// @flow
+import PageWrapper from '@weco/common/views/components/PageWrapper/PageWrapper';
+import {spacing, grid} from '@weco/common/utils/classnames';
+
+const ProgressPage = () => (
+  <div className={`row ${spacing({s: 5}, {padding: ['top']})}`}>
```
Comment (davidpmccormick) [inline catalogue/webapp/pages/progress.js]: Could/should this Christmas tree of divs be replaced with one of our Layout components?

File: catalogue/webapp/pages/work.js [hunk 2]
```diff
@@ -413,7 +413,7 @@ export const WorkPage = ({
                     ])}>
                       <Icon name='underConstruction' extraClasses='margin-right-s2' />
                       <p className={`${font({s: 'HNL5', m: 'HNL4'})} no-margin`}>
-                        We’re improving the information on this page. <a href='/progress'>Find out more</a>.
+                        We’re improving the information on this page. <a href='/works/progress'>Find out more</a>.
```
Comment (davidpmccormick) [inline catalogue/webapp/pages/work.js]: Were the progress notes supposed to be specific to works? I'm not sure that they were (even though they obviously are for the time being).

File: catalogue/webapp/pages/progress.js (hunk 1 above)
Comment (jamesgorrie) [reply catalogue/webapp/pages/progress.js]: Probably, the copy pasting, reformating nunjucks => jsx has just left me really bored of it.

File: catalogue/webapp/pages/work.js (hunk 2 above)
Comment (jamesgorrie) [reply catalogue/webapp/pages/work.js]: Let's say that these progress notes are, we can generify them if we move in that direction.

File: catalogue/webapp/pages/progress.js (hunk 1 above)
Comment (davidpmccormick) [reply catalogue/webapp/pages/progress.js]: Testify!

**Pull request merged**